ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -303,25 +298,7 @@ impl Module {
     ///
     /// Panics if the index is out of bounds.
     pub(crate) fn global_decl(&self, idx: GlobalDeclIdx) -> &GlobalDecl {
-        &self.global_decls[idx]
-    }
-
-    /// Get the index of a global, inserting it into the global declaration table if necessary.
-    ///
-    /// `aot_idx` is the [aot_ir::GlobalDeclIdx] index of the global in the AOT module. This is
-    /// needed to find the global variable's address in the global variable pointers array.
-    pub(crate) fn global_decl_idx(
```

Comment:
Surely by not using an AOT index here you are looking up the wrong function address?

Reply:
Note that `_aot_idx` was not used in this function!

Reply:
Ah right. The AOT index is inside `GlobalDecl`. Yeah I agree, this is just noise then.